### PR TITLE
Link accounts

### LINKED ISSUE #104: Should profile pages include social links?
image

### PR DESCRIPTION
**Pull request by schneider-clarisse**
Switch the linkaccounts to just setting facebook/twitter/linkedin things that prepopulate

fixes #104 and #105

### CODE REVIEW

**Comment by larajanecka** [comment]
This is rebased off the styling branch right? You cool with me waiting for that to go in before I review this?

**Comment by schneider-clarisse** [comment]
Yes (y)

**Comment by schneider-clarisse** [comment]
@larajanecka styling merged

**Comment by larajanecka** [comment]
on it

File: src/js/pages/attendee/Profile.jsx [hunk 1]
```diff
@@ -136,13 +144,31 @@ export default class Profile extends React.Component {
...
         { (myProfile || this.state.description.length > 0) &&
           <Input type='text' label='About you' name='description' value={this.state.description} onChange={this.handleChange.bind(this, 'description')} disabled={!myProfile}/>
         }
+        { myProfile &&
+          <div className="socialMediaInput">
+            <div className="prefix">facebook.com/</div>
+            <Input type='text' label='Facebook' name='facebook' value={this.state.facebook} onChange={this.handleChange.bind(this, 'facebook')} disabled={!myProfile}/>
+          </div>
+        }
+        { myProfile &&
```
Comment (larajanecka) [inline src/js/pages/attendee/Profile.jsx]: could these all be combined in one block?

**Comment by larajanecka** [comment]
image
This looks a bit funny to me.

**Comment by schneider-clarisse** [comment]
@larajanecka fixed

**Comment by larajanecka** [comment]
Oh do I just fail at pulling?

**Review by larajanecka — APPROVED** [review]

**Comment by larajanecka** [comment]
kk lgtm merge at will

File: src/js/pages/attendee/Profile.jsx (hunk 1 above)
Comment (schneider-clarisse) [reply src/js/pages/attendee/Profile.jsx]: gg

**Pull request merged**